cognite/neat/rules/models/_rules/dms_architect_rules.py

```diff
@@ -953,16 +953,19 @@ def as_information_architect_rules(
             updated=dms.updated or updated or datetime.now(),
         )
 
-        classes: list[InformationClass] = [
+        classes = [
             InformationClass(
-                # we do not want version in class as we use URI for the class
+                # we do not want a version in class as we use URI for the class
                 class_=view.class_.as_non_versioned_entity(),
                 description=view.description,
                 parent=[
-                    # we do not want version in class as we use URI for the class
-                    ParentClassEntity(prefix=implented_view.prefix, suffix=implented_view.suffix)
-                    for implented_view in view.implements or []
+                    # we do not want a version in class as we use URI for the class
+                    ParentClassEntity(prefix=implemented_view.prefix, suffix=implemented_view.suffix)
+                    # We only want parents in the same namespace, parent in a different namespace is a reference
+                    for implemented_view in view.implements or []
+                    if implemented_view.prefix == view.class_.prefix
                 ],
+                reference=self._get_class_reference(view),
```

Comment:
Unlike parent class where we do not keep version inf, here we do ! great ! reference concept is paying dividens!